src/Item/Item.schema.ts

```diff
@@ -8,8 +8,12 @@ export const itemSchema = Object.freeze({
   properties: {
     id: { type: 'string', format: 'uuid' },
     urn: { type: ['string', 'null'] },
-    name: { type: 'string', maxLength: 32 },
-    description: { type: ['string', 'null'], maxLength: 64 },
+    name: { type: 'string', maxLength: 32, pattern: '^[^:]*$' },
+    description: {
+      type: ['string', 'null'],
+      maxLength: 64,
+      pattern: '^[^:]*$',
```

Comment:
shouldn't these have a `+` instead of an `*` ? or at least a minLength key?

Reply:
The only check that the builder has is this one, there's sadly no check about the length. We could add one, but we'll be potentially introducing a breaking change.